pkg/util/validation/stack_test.go

```diff
@@ -9,6 +9,43 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestValidateStackName(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		title     string
+		stackName string
+		error     string
+	}{
+		{"empty", "", "a stack name may not be empty"},
+		{"valid", "foo", ""},
+		{
+			"slash", "foo/bar",
+			"a stack name may only contain alphanumeric, hyphens, " +
+				"underscores, or periods: invalid character '/'" +
+				" at position 3",
+		},
```

Comment:
would you mind adding another test for the failure case where `failure = 0` first character is invalid)?

Reply:
Sure.